test/extended/deployments/deployments.go

```diff
@@ -379,7 +379,7 @@ var _ = g.Describe("deploymentconfigs", func() {
 			out, err := oc.Run("rollout").Args("history", resource).Output()
 			o.Expect(err).NotTo(o.HaveOccurred())
 			g.By(fmt.Sprintf("checking the history for substrings\n%s", out))
-			o.Expect(out).To(o.ContainSubstring("deploymentconfigs \"deployment-simple\" history viewed"))
+			o.Expect(out).To(o.ContainSubstring("deploymentconfigs \"deployment-simple\""))
```

Comment:
this is fine, the upstream PR ended up not including it: https://github.com/kubernetes/kubernetes/pull/27267